polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -259,6 +259,48 @@ $loading-panel-height: 53px;
     }
   }
 
+  &.TableRow-child {
+    cursor: default;
+    // stylelint-disable -- Polaris component custom properties
+    --pc-index-table-checkbox-offset-left: var(--p-space-4);
+    --pc-index-table-checkbox-width: var(--p-space-5);
+    --pc-index-table-checkbox-border-width: var(--p-border-width-2);
+    --pc-index-table-checkbox-child-offset: calc(
+      var(--pc-index-table-checkbox-offset-left) +
+        var(--pc-index-table-checkbox-border-width) +
+        var(--pc-index-table-checkbox-width)
+    );
+
+    #{$se23} & {
+      // stylelint-disable -- Polaris component custom properties
+      --pc-index-table-checkbox-offset-left: var(--p-space-3);
```

Comment:
When tophatting polaris uplift, the cell content does not align with the cell table header also resulting in different spacing amount between the checkbox and table header vs. checkbox and table row content

https://github.com/Shopify/polaris/assets/15054990/ca5ec158-5ead-43ff-9432-9439b70a3555



<img width="629" alt="Screenshot 2023-09-25 at 3 58 08 PM" src="https://github.com/Shopify/polaris/assets/15054990/1f817de0-dcf8-4dcf-9c39-3f322911528e">

Reply:
With @shopify/polaris@12.0.0 [has shipped](https://shopify.slack.com/archives/C4Y8N30KD/p1696832186682759) we don't need to duplicate styles and use that flag anymore 🎉